### ISSUE TITLE
sync mode broken inside multiremote customCommands

### ISSUE THREAD

**Issue by js-kyle** [webdriverio/webdriverio#4430]
**Environment (please complete the following information):**
 - **WebdriverIO version:** 5.12.5
 - **Mode:** WDIO Testrunner
 - **If WDIO Testrunner, running sync/async:** sync
 - **Node.js version:** 10.14.1
 - **NPM version:** 6.4.1
 - **Browser name and version:** Chrome 76
 - **Platform name and version:** MacOS 10.14.4
 - **Additional wdio packages used (if applicable):** 
```
    "@wdio/cli": "^5.12.4",
    "@wdio/devtools-service": "^5.12.3",
    "@wdio/local-runner": "^5.12.4",
    "@wdio/mocha-framework": "^5.12.1",
    "@wdio/spec-reporter": "^5.12.1",
    "@wdio/sync": "^5.12.3",
    "wdio-chromedriver-service": "^5.0.2",
    "webdriverio": "^5.12.5"
```
**Config of WebdriverIO**
```
exports.config = {
  port: 9515 ,
  path: '/',
  sync: true,
  maxInstances: 1,
  services: [
    'chromedriver',
  ],
  capabilities: {
    browserA: {
      capabilities: {
        browserName: 'chrome'
      }
    },
    browserB: {
      capabilities: {
        browserName: 'chrome'
      }
    }
  }
}
```

**Describe the bug**
It appears that when using customCommands in multiremote mode, that the sync feature does not work correctly.

**To Reproduce**

These commands work, and log the HTML as expected:
```
  it('working demo', function() {
    browserA.url('https://google.com');
    let el = browserA.$('body');
    console.log(el.getHTML()); // this logs HTML
  });
```
**The same commands as above, do not resolve the promises when inside a customCommand:**
```
  it('broken demo', function() {

    browserA.addCommand('testCommand', function() {
      browserA.url('https://google.com');
      let el = browserA.$('body');
      console.log(el); This logs Promise { <pending> }
      console.log(el.getHTML()); TypeError: el.getHTML is not a function
    });
    browserA.testCommand();
  });
```

**Expected behavior**
Commands should return the value, rather than a pending Promise

**Additional context**
The problem is not seen when running with a single browser/capability. The issue can only be reproduced inside customCommands.

If the customCommand is changed to a async function which uses `await`, the values are resolved, which leads me to believe it is an issue to do with @wdio/sync.

Many thanks for this project - happy to be upgrading to the improved v5, just working on this last block!

**Comment by christian-bromann** [comment]
fixed in #4438

**Comment by js-kyle** [comment]
@christian-bromann awesome 👏
